Remove ATen test that fails to find sleef.h during build on Intel systems when -march=native is set
Fix from https://github.com/pytorch/pytorch/issues/26555#issuecomment-533801098
Patch created by Simon Branford of the BEAR Software team at the University of Birmingham
--- aten/src/ATen/test/CMakeLists.txt.orig	2019-11-17 10:59:54.368096000 +0000
+++ aten/src/ATen/test/CMakeLists.txt	2019-11-17 11:00:16.478435000 +0000
@@ -27,7 +27,6 @@
   ${CMAKE_CURRENT_SOURCE_DIR}/quantized_test.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/extension_backend_test.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/xla_tensor_test.cpp
-  ${CMAKE_CURRENT_SOURCE_DIR}/tensor_iterator_test.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/cpu_generator_test.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/pow_test.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/reduce_ops_test.cpp)
